packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -83,6 +83,10 @@ class BuildIOSFrameworkCommand extends BuildSubCommand {
         defaultsTo: true,
         hide: !verboseHelp,
       )
+      ..addFlag('include-dsym',
```

Comment:
Is there any reason to not include it?  I'd rather just include it instead of an undiscoverable flag and let Xcode filter it as needed on the embedding side.

Reply:
Actually, I'm not sure if Flutter team intended to exclude dSYM or not, so I added flag just in case, I'm going to remove it.